docs/bigquery/snippets.py

```diff
@@ -159,27 +159,43 @@ def test_update_dataset_multiple_properties(client, to_delete):
     # [START update_dataset_multiple_properties]
     assert dataset.description == ORIGINAL_DESCRIPTION
     assert dataset.default_table_expiration_ms is None
-    entry = bigquery.AccessEntry(
-        role='READER', entity_type='domain', entity_id='example.com')
-    assert entry not in dataset.access_entries
     ONE_DAY_MS = 24 * 60 * 60 * 1000  # in milliseconds
     dataset.description = UPDATED_DESCRIPTION
     dataset.default_table_expiration_ms = ONE_DAY_MS
-    entries = list(dataset.access_entries)
-    entries.append(entry)
-    dataset.access_entries = entries
 
     dataset = client.update_dataset(
         dataset,
-        ['description', 'default_table_expiration_ms', 'access_entries']
+        ['description', 'default_table_expiration_ms']
     )  # API request
 
     assert dataset.description == UPDATED_DESCRIPTION
     assert dataset.default_table_expiration_ms == ONE_DAY_MS
-    assert entry in dataset.access_entries
     # [END update_dataset_multiple_properties]
 
 
+def test_update_dataset_access(client, to_delete):
+    """Update a dataset's metadata."""
+    DATASET_ID = 'update_dataset_access_%d' % (_millis(),)
```

Comment:
Since we're here, make it `dataset_id` and just use `.format()` instead of `%` formatting?

(If there are no other changes requested for this PR, don't bother.)

Reply:
Done.